neuralmonkey/runners/label_runner.py

```diff
@@ -30,6 +33,7 @@ def get_executable(self, compute_losses=False, summaries=True):
         fetches["input_mask"] = self._decoder.encoder.input_sequence.mask
 
         return LabelRunExecutable(self.all_coders, fetches,
+                                  num_sessions,
```

Comment:
bud to zařízni za tim all_coders, nebo to pofrkej tak aby to bylo za sebou jak se to vejde

Reply:
+ type hints